generators/server/templates/src/main/java/package/_Application.java

```diff
@@ -54,6 +55,8 @@
     @Inject
     private Environment env;
 
+    public static ApplicationContext context;
```

Comment:
Using an app server is clearly a bad idea, but we can't force people :-)
Not sure about it, but could use JMX to get it?
Otherwise you won't find any "static" in Spring, because of those classloader issues (they also used to cause memory leaks in some app servers, which were not dereferencing correctly static fields)